setup/lidarr.iss

```diff
@@ -6,7 +6,7 @@
 #define AppURL "https://lidarr.audio/"
 #define ForumsURL "https://forums.lidarr.audio/"
 #define AppExeName "Lidarr.exe"
-#define BaseVersion "0.6.2"
+#define BaseVersion "0.7.0"
```

Comment:
We can make this GetEnv(MAJORVERSION) I think

Reply:
That would be cool, less places to have to change the better